rel/relx.config.src

```diff
@@ -9,6 +9,7 @@ Filter = ['rabbitmq_codegen'
           ,'.settings'
           ,'skel'
           ,'parse_trans'
+          ,'sup'
```

Comment:
@jamhed so instead of fixing it you just remove it ? whatif another app fails to build for release ? would you remove it ?